.github/workflows/ci_master.yaml

```diff
@@ -24,12 +24,13 @@ jobs:
         run: |
           touch .env
           echo NX_RIVET_KEY=${{ secrets.RIVET_KEY }}  >> .env
+          echo CERAMIC_NETWORK=${{ secrets.CERAMIC_NETWORK }}  >> .env
```

Comment:
I'd love a brief TLDR at some point when you have bandwidth of what you had to do to fix this -- it looks like you had to add a specific CERAMIC_NETWORK key for this to work?

Mainly just curious!

Reply:
Yes, definitely